main.C

```diff
@@ -240,11 +240,12 @@ public:
 
     if (item->Key().size()) {
       const string &key = item->Key();
+      const string &value = item->Value();
       const UINT32 &flags = item->Flags();
 
-      if (((flags & APE_FLAG_READONLY) == APE_FLAG_READONLY) && !force) {
```

Comment:
how can a user change the readonly flag now?

what if some user relies on this?

Reply:
This still works. I fixed a bug where the item type could be changed successfully (e.g. -p to -r flag) at the same time as the value leading to the ro flag getting cleared. And added unit testing to check for this occurence.